src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppAppRunner/Generated/Recipe.cs

```diff
@@ -44,20 +46,61 @@ public Recipe(Construct scope, IRecipeProps<Configuration> props)
             ConfigureAppRunnerService(props);
         }
 
+        private void ConfigureVpc(Configuration settings)
+        {
+            if (settings.VPCConnector.UseVPCConnector)
+            {
+                if (settings.VPCConnector.CreateNew)
+                {
+                    if (settings.VPCConnector.CreateNewVpc)
+                    {
+                        AppVpc = new Vpc(this, nameof(AppVpc), InvokeCustomizeCDKPropsEvent(nameof(AppVpc), this, new VpcProps
+                        {
+                            MaxAzs = 2
+                        }));
+                    }
+                }
+            }
+        }
+
         private void ConfigureVPCConnector(Configuration settings)
         {
             if (settings.VPCConnector.CreateNew)
             {
-                if (settings.VPCConnector.Subnets.Count == 0)
-                    throw new InvalidOrMissingConfigurationException("The provided list of Subnets is null or empty.");
+                if (settings.VPCConnector.CreateNewVpc)
+                {
+                    ConfigureVpc(settings);
+
+                    if (AppVpc == null)
+                        throw new InvalidOperationException($"{nameof(AppVpc)} has not been set.");
+
+                    var vpcSubnets = new SortedSet<string>();
```

Comment:
Never combine private and public subnets. According to the App Runner developer guide we should be using private subnets. https://docs.aws.amazon.com/apprunner/latest/dg/network-vpc.html#network-vpc.subnet

Reply:
Done